File: Plugins/Aspose_Cells_Java_for_PHP/src/aspose/cells/WorkingWithFiles/UtilityFeatures/Excel2PdfConversion.php
<?php

namespace Aspose\Cells\WorkingWithFiles\UtilityFeatures;

use com\aspose\cells\Workbook as Workbook;
use com\aspose\cells\SaveFormat as SaveFormat;


class Excel2PdfConversion {

    public static function run($dataDir=null)
    {

        $saveFormat = new SaveFormat();

        $workbook = new Workbook($dataDir . "Book1.xls");

        //Save the document in PDF format
        $workbook->save($dataDir . "OutBook1.pdf", $saveFormat->PDF);

        // Print message
        echo "\n Excel to PDF conversion performed successfully.";
    }

}

File: Examples/src/main/java/com/aspose/cells/examples/loading_saving/Excel2PDFConversion.java
package com.aspose.cells.examples.loading_saving;

import com.aspose.cells.*;
import com.aspose.cells.examples.Utils;

public class Excel2PDFConversion {

	public static void main(String[] args) throws Exception {

		// The path to the documents directory.
		String dataDir = Utils.getSharedDataDir(Excel2PDFConversion.class) + "loading_saving/";

		Workbook workbook = new Workbook(dataDir + "Book1.xls");

		// Save the document in PDF format
		workbook.save(dataDir + "E2PDFC-out.pdf", SaveFormat.PDF);

		// Print message
		System.out.println("Excel to PDF conversion performed successfully.");

	}
}
